modules/end-user-guide/pages/mounting-volumes-secrets-and-configmaps.adoc

```diff
@@ -0,0 +1,16 @@
+:navtitle: Mounting Volumes, Secrets and ConfigMaps
+:keywords: user-guide, configuring, user, configmaps, secrets, volumes, mounting, mount
+:page-aliases: 
+
+[id="mounting-volumes-secrets-and-configmaps_{context}"]
+= Mounting Volumes, Secrets and ConfigMaps
+
+Users are able to mount their existing Persistent Volume Claims (PVC), Secrets and ConfigMaps into their {devworkspace} containers. 
+
+The mounting process uses the standard {kubernetes} mounting mechanism. Mounting to the {devworkspace} containers requires additional labels and annotations applied to existing resources.
+
+Resources are mounted when starting a new {devworkspace} or restarting an existing one.
```

Comment:
I'm not able to relate this to the UX and UI.
Is `{devworkspace}` a workspace?

Reply:
The user would create or find a secret/configmap that contains data that they would want to mount to their devworkspace containers

Reply:
I've changed this to `starting a new workspace` in my latest commit.